Assets/Scripts/Spawnables/Seeker.cs

```diff
@@ -83,42 +87,40 @@ private void RotateTowardTarget(Vector3 dir)
             
             // Figure out amount to move this frame
             var step = currentSpeed * Time.deltaTime;
-            float animSpeed = 0;
-            
-            
-            if (Time.time % (idleDuration + seekDuration) < idleDuration)
+            var animSpeed = 0.0f;
+
+            if ((Time.time + _jitter) % (idleDuration + seekDuration) < idleDuration)
```

Comment:
I still get some ghouls synched... what if we randomised the _jitter after every move?

Reply:
I will add this to the polish list :D